04EstruturasDeControle/test/EstruturasDeControleExercicioTest.java

```diff
@@ -1,8 +1,62 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
 public class EstruturasDeControleExercicioTest {
     // - Funcionalidade - Retornar nome do mês do ano para qualquer número
     // do mês passado. Caso seja um número inválido, retornar message de erro
 
-    /* -----------------
-     * TODO
-     * ----------------- */
+    String meses(int numeroMes) {
```

Comment:
[Opcional]

O nome do método acaba dando a entender que ele retorna mais de um mês, em vez de um unico mês baseado no valor inteiro. Talvez seja interessante alterar o nome do método. Aqui vai uma sugestão:
```suggestion
    String mes(int numeroMes) {
```